scripts/unocommands.py

```diff
@@ -407,13 +452,19 @@ def writeTranslations(onlineDir, translationsDir, strings):
         parsed = parseUnocommandsJS(onlineDir)
         processedCommands = set(parsed.keys())
     else:
-        written = writeUnocommandsJS(onlineDir, lofficeDir, menuCommands, contextCommands, toolbarCommands)
+        written = writeUnocommandsJS(onlineDir, lofficeDir, menuCommands,
+                                     contextCommands, toolbarCommands)
         processedCommands = set(written.keys())
 
     # check that we have translations for everything
-    dif = (menuCommands | contextCommands | toolbarCommands) - processedCommands
+    dif = (menuCommands | contextCommands or
+           toolbarCommands) - processedCommands
+
     if len(dif) > 0:
-        sys.stderr.write("ERROR: The following commands are not covered in unocommands.js, run scripts/unocommands.py --update:\n\n.uno:" + '\n.uno:'.join(dif) + "\n\n")
+        sys.stderr.write("ERROR: The following commands",
```

Comment:
You can't slice the input like this. write() takes only 1 argument, but now you are giving 4.

Reply:
I made the change thank you : )